src/modules/ndb_redis/redis_client.c

```diff
@@ -1061,3 +1067,57 @@ int redis_count_err_and_disable(redisc_server_t *rsrv)
 	}
 	return 0;
 }
+
+void print_redis_reply(int log_level, redisReply *rpl,int offset)
+{
+	int i;
+	char padding[MAXIMUM_NESTED_KEYS + 1];
+
+	if (redis_get_debug_level() < log_level)
+		return;
+
```

Comment:
I think that instead of the above IF condition you can use:

```
if(!is_printable(log_level))
```

so redis_get_debug_level() is not needed at all.